### PR TITLE
Tighten-up eslint config

### PR DESCRIPTION
**Pull request by jeddy3**
Getting things ready in case other people want to help out...

I've enabled a couple of rules for consistent whitespace in the code e.g. after the `function` keyword and within parentheses etc

The current code in master is nice and consistent, so no changes were needed there. :)

Ok with you guys?

Ref #15

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -26,5 +19,35 @@ rules:
...
+
+  # consistent whitespace after keywords
+  space-after-keywords: [2, "always"]
+
+  # consistent whitespace in parens
+  space-in-parens: [2, "never"]
+
+  # consistent whitespace in brackets (disabled as buggy)
+  # space-in-brackets: [2, "always"]
+
```
Comment (MoOx) [inline .eslintrc]: If buggy, please add as comment url of the related issue so we think to check/subscribe :)

**Comment by MoOx** [comment]
Except my comment, lgtm

**Comment by jeddy3** [comment]
I've added the URL to the issue :)

[babel/babel-eslint#87](https://github.com/babel/babel-eslint/issues/87)

File: .eslintrc [hunk 2]
```diff
@@ -26,5 +19,36 @@ rules:
...
+  space-after-keywords: [2, "always"]
+
+  # consistent whitespace before blocks
+  space-before-blocks: [2, "always"]
+
+  # consistent whitespace before function paren
+  space-before-function-paren: [2, {"anonymous": "always", "named": "never"}]
+
+  # consistent whitespace in brackets
+  # disabled as possibly buggy: https://github.com/babel/babel-eslint/issues/87
```
Comment (hzoo) [inline .eslintrc]: Ok I tested and this is fixed in eslint master. You guys just got to wait for the next release or change the `package.json` to reference that commit.

Comment (jeddy3) [inline .eslintrc]: Thank you!

Comment (MoOx) [inline .eslintrc]: thanks for letting us know @hzoo !

**Pull request merged**